AutoStack/Main.js

```diff
@@ -0,0 +1,62 @@
+const execSync = require("child_process").execSync;
+const http = require("http");
+const MineShell = require("./MineShell.js");
```

Comment:
don't we want to take the suffix out from the require statement?

Reply:
yes! I need to rebase to get those changes